src/python/WMCore/MicroService/MSPileup/MSPileup.py

```diff
@@ -27,7 +27,8 @@ def __init__(self, msConfig, **kwargs):
         # Get the RSE expression for Disk RSEs from the configuration
         self.diskRSEExpr = msConfig.get("rucioDiskExpression", "")
 
-    def status(self):
+    @staticmethod
+    def status():
```

Comment:
Just for my education, I guess you made this change just because this method does not depend on any of the object data, right? Is there any other motivation/benefit that I might be missing?

Reply:
I believe so. I changed this because it came up in Pylint